WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -80,20 +80,25 @@ public WalletViewModel(bool receiveDominant)
 			advancedAction.Items.Add(infoTab);
 			advancedAction.Items.Add(buildTab);
 
-			// Open and select tabs.
+			// Open tabs.
 			sendTab?.DisplayActionTab();
-			if (receiveDominant)
+			receiveTab.DisplayActionTab();
+			coinjoinTab.DisplayActionTab();
+			historyTab.DisplayActionTab();
```

Comment:
Why is this?

Reply:
I did that to open the tabs in the same order all the time, I think this is better UX:
Send | Receive | CoinJoin | History

Otherwise the tabs will be in different order depending on which one was last active:
Send | Receive | CoinJoin | History
Send | CoinJoin | History | Receive
Send | Receive | History | CoinJoin